deploy/007-deploy-mock-tokens.ts

```diff
@@ -26,7 +26,5 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 };
 
 func.tags = ["MockTokens"];
-// These are mock tokens
-func.skip = async hre => hre.network.name !== "hardhat";
```

Comment:
Will not we deploy the mock tokens to mainnet with this change?

Reply:
Tokens are defined in `tokenConfig` mapping, `isMock` should be false for mainnet networks